schema/transportation/segment.yaml

```diff
@@ -392,6 +417,27 @@ properties:
           description: direction of travel on the target that pertains to the relation
           "$ref": "#/$defs/propertyDefinitions/direction"
       required: [segmentId]
+    compassDirection:
+      description: >-
+        Four cardinal and four ordinal direction
+      type: string
+      enum:
+        - East
+        - West 
+        - North
+        - South 
+        - NorthEast
+        - NorthWest 
+        - SouthEast
+        - SouthWest 
```

Comment:
The schema naming standard is `camelCase`, so this is `northEast`, `northWest`, *etc.*

Incidentally this probably isn't the first time I've made a comment like this....